ci-operator/step-registry/ipi/conf/openstack/generateconfig/ipi-conf-openstack-generateconfig-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+#Set necessary variables
+CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
+OS_CLOUD=$(<"${SHARED_DIR}"/OS_CLOUD)
+OPENSTACK_EXTERNAL_NETWORK=$(<"${SHARED_DIR}"/OPENSTACK_EXTERNAL_NETWORK)
+OPENSTACK_COMPUTE_FLAVOR=$(<"${SHARED_DIR}"/OPENSTACK_COMPUTE_FLAVOR)
+LB_FIP_IP=$(<"${SHARED_DIR}"/LB_FIP_IP)
+BASE_DOMAIN=$(<"${SHARED_DIR}"/BASE_DOMAIN)
+CONFIG_TYPE=$(<"${SHARED_DIR}"/CONFIG_TYPE)
+
+PULL_SECRET=$(<"${CLUSTER_PROFILE_DIR}"/pull-secret)
+SSH_PUB_KEY=$(<"${CLUSTER_PROFILE_DIR}"/ssh-publickey)
+
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+if [[ "${CONFIG_TYPE}" == "minimal" ]]; then
+cat >> "${CONFIG}" << EOF
+apiVersion: v1beta4
+        baseDomain: ${BASE_DOMAIN}
+        controlPlane:
+          name: master
+        metadata:
+          name: ${CLUSTER_NAME}
+        platform:
+          openstack:
+            cloud:            ${OS_CLOUD}
+            externalNetwork:  ${OPENSTACK_EXTERNAL_NETWORK}
+            computeFlavor:    ${OPENSTACK_COMPUTE_FLAVOR}
+            lbFloatingIP:     ${LB_FIP_IP}
+        pullSecret: >
```

Comment:
pull secret and sshkey are set in `ipi-conf` step

Reply:
yes, the intention here is to allow for multiple types of install config.  My understanding is that the environment variables set in other steps will not propage accross steps.  only variables declared in "env" sections of refs, chains, or workflows are available everywhere.  so we read it here. For now it seems redundant but the number of different install configs will grow to 4 or five.

Reply:
There might be a way to concat the install-config.yaml created ini ipi-conf with the one generated here. Will look into that.